recipes/oso/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "oso" %}
+{% set build_number = 0 %}
+{% set version = "0.26.1" %}
+{% set min_rust = "rust >=1.46" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/osohq/oso/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 412b62ca46b649cd2b0ddfd9027180eeda9aa8d4f059da44978fa56ac103e90a
+
+build:
+  number: {{ build_number }}
+  script:
+    - bash {{ RECIPE_DIR }}/build_oso.sh
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ min_rust }}
+  host:
+    - jq
+    - make
+    - clangdev                               # [not osx]
+    - openssl
+    - python
+    - pip
+    - cffi >=1.0.0
+  run:
+    - python
+    # fails pip check: https://github.com/conda-forge/importlib_metadata-feedstock/issues/88
+    - importlib-metadata !=4.7.1
+    - cffi >=1.0.0
+
+test:
+  source_files:
+    - languages/python/oso/tests
+  imports:
+    - oso
+    - polar
+  commands:
+    - cd languages/python/oso
+    - pip check
+    - pytest -vv tests
+  requires:
+    - pip
+    - pytest-cov
+    - ruamel_yaml
+    - jq
+    - sqlalchemy
+
+about:
+  home: https://github.com/osohq/oso
+  summary: 'Oso is a batteries-included framework for building authorization in your application.'
+  license: Apache
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
As this bundles a bunch of third-party stuff in the rust static binary, you'll want to use [cargo-bundle-licenses](https://github.com/sstadick/cargo-bundle-licenses) to capture all of the built static binary licenses. There are many [examples in other feedstocks](https://github.com/search?q=org%3Aconda-forge+cargo-bundle-licenses&type=code).

As annoying as this is, it's _far far better_ than the bad old days of curating them by hand.